covasim/webapp/cova_app.css

```diff
@@ -121,6 +121,7 @@ fieldset > legend {
     display: block;
     position: relative !important;
     width: initial;
+    /*overflow-y: scroll; CK: for later use */
```

Comment:
That comment isn't meaningful to anyone and will likely get deleted by accident since it doesn't have any context.

Reply:
No worries if it does, but hopefully
https://github.com/InstituteforDiseaseModeling/covasim/issues/201
will get done soon and we can have actual overflow-y instead of it being commented out.